### DISCUSSION TITLE
**title** in header scrolling

### DISCUSSION THREAD

**Discussion by hudsondiegof** [squidfunk/mkdocs-material#5886] [Q&A] [answered]
when I scroll down and I have a main heading that is in bold. Asterisks ** appear in the header.

I would like to leave my title in bold and that the ** does not appear in the header.

`# **title**`

![image](https://github.com/squidfunk/mkdocs-material/assets/83621099/82b05e4c-19a9-4ec6-a6b2-ff3ab6f92a3a)

**Comment by squidfunk** [comment]
I'm sorry, but I can reproduce the problem, i.e., I'm not sure what combination of configuration and writing you're using to observe this behavior. Please provide a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/), so we know what we're talking about.

**Reply by hudsondiegof** [reply, marked as answer]
The problem only happens on my local server. When I pushed to git, the problem was resolved.

So I don't need a solution anymore. What matters is the result on github pages.

And it's working fine.

Thanks.
